.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+
+end_of_line = lf
+insert_final_newline = true
```

Comment:
Oh, nice) It looks like those files belong to the config of IDE) Should they be in the final PR?

Reply:
Yes, it should. editorconfig standardizes the editor configuration for any (most famous) IDE.